pkg/operators/ebpf/ebpf.go

```diff
@@ -178,6 +181,14 @@ func (i *ebpfInstance) loadSpec() error {
 	if err != nil {
 		return fmt.Errorf("loading spec: %w", err)
 	}
+
+	// For USDT programs, we need to add the BPF_TRACE_UPROBE_MULTI flag
```

Comment:
cilium/ebpf automatically adds the MULTI flag when the section is named "uprobe.multi":
https://github.com/cilium/ebpf/blob/v0.15.0/elf_sections.go#L21-L22

Unfortunately, it does not have a similar section name for usdt. Do you think that should be something to improve in cilium/ebpf and libbpf? In this way, you would not need to manually change the AttachType and you would not need to redefine the constant BPF_TRACE_UPROBE_MULTI.

Meanwhile, this is fine for this PR, since there is no other choices with the current cilium/ebpf.

Reply:
I don't think this is cilium/ebpf's issue. Here I add the `BPF_TRACE_UPROBE_MULTI` flag, only because I'm using a trick to deal with USDT tracepoints which are with the same name.

For example, there are many `trace_gc` USDT probes in Libjvm, and I need to attach to all of them. Using `BPF_TRACE_UPROBE_MULTI` will make it faster. Also, I don't want the `uprobetracer` be aware of this and support nested Links. Using `BPF_TRACE_UPROBE_MULTI` will merge many `ebpf.Link`s into one, and I think that's more elegant.